utils/hare-bootstrap

```diff
@@ -353,20 +366,22 @@ wait4 ${pids[@]-}
 agents_nr=$(( ${#pids[@]} + 1 ))
 
 # Waiting for the Consul agents to get ready...
-count=1
 while (( $(get_ready_agents_nr) != $agents_nr )); do
-    if (( $count > 5 )); then
-        echo 'Some agent(s) failed to start in due time:' >&2
-        diff <(get_ready_agents | sort) \
-             <(get_all_nodes | awk '{print $1}' | sort) | sed 1d >&2
+    echo 'Consul not ready on node :'
```

Comment:
I think instead of checking in every iteration, which would generate more logs, let's keep the 5 secs timeout and check_consul_failure every 5 secs as earlier, just don't exit after 5 secs. Only if check_consul_failure returns non-zero (i.e. consul failed to start) the script must exit with an error.

Reply:
@mssawant  Done